### PR TITLE
Linking to updated overview PDF

### PR DESCRIPTION
**Pull request by Hasmin-AC**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/thank-you/download-platform-overview.njk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Thank you for your interest! 
-subtitle: "<a href='https://26586079.fs1.hubspotusercontent-eu1.net/hubfs/26586079/FlowFuse%20Platform%20Overview.pdf'>Click here</a> to access the The FlowFuse Paltform Overview"
+subtitle: "<a href='https://26586079.fs1.hubspotusercontent-eu1.net/hubfs/26586079/FlowFuse%20Platform%20Overview-October%202024.pdf'>Click here</a> to access the The FlowFuse Platform Overview"
```
Comment (Yndira-E) [inline src/thank-you/download-platform-overview.njk:3]: Hi Hasmin, I noticed you used one of the PDF files I had shared earlier, but it wasn’t the final version—I was just checking quality with that one. Would you mind downloading the latest PDF from Google Docs to ensure it’s the most up-to-date? Thank you!

Comment (Hasmin-AC) [reply src/thank-you/download-platform-overview.njk:3]: Did you have a PDF? when I converted [this](https://docs.google.com/document/d/1VtLQMZvvGDqTjiplUoUtdLwEeS9W5F7Y-42N2zGmRFc/edit?tab=t.0), it looks pretty much the same as the one I uploaded.

Comment (Yndira-E) [reply src/thank-you/download-platform-overview.njk:3]: I know there were tweaks after that, the only one that I can recall that would be obvious is the order of text/image in the section where two points share a screenshot

Comment (Hasmin-AC) [reply src/thank-you/download-platform-overview.njk:3]: replaced the file on the same location.

**Pull request merged**